content/en/continuous_integration/setup_tests/python.md

```diff
@@ -313,16 +313,17 @@ DD_SERVICE=my-python-app DD_ENV=ci pytest --ddtrace --ddtrace-patch-all
 
 ### Adding custom tags to tests
 
-You can add custom tags to your tests by using the current active span:
+You can add custom tags to your tests by using the declaring `ddspan` as argument to your test:
 
 ```python
 from ddtrace import tracer
 
-// inside your test
-span = tracer.current_span()
-span.set_tag("test_owner", "my_team")
-// test continues normally
-// ...
+# Declare `ddspan` as argument to your test
+def test_simple_case(ddspan):
+    # Set your tags
+    ddspan.set_tag("test_owner", "my_team")
+    # test continues normally
+    # ...
```

Comment:
```suggestion
# Declare `span` as argument to your test
def test_simple_case(span):
    # Set your tags
    span.set_tag("test_owner", "my_team")
    # test continues normally
    # ...
```

Reply:
`ddspan` is a fixture:
https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/contrib/pytest/plugin.py#L124

We should use `ddspan` :+1: